README.adoc

```diff
@@ -4002,6 +4002,30 @@ format('Hello, %{name}', name: 'John')
 format('Hello, %<name>s', name: 'John')
 ----
 
+=== Long Strings [[heredoc-long-strings]]
+
+Break long strings into multiple lines but don't concatenate them with `+`.
+If you want to add newlines, use `Heredoc`. Otherwise use `\`:
```

Comment:
I think that backquotes are unnecessary because `Heredoc` is not a code.

```diff
- If you want to add newlines, use `Heredoc`. Otherwise use `\`:
+ If you want to add newlines, use heredoc. Otherwise use `\`:
```